### DISCUSSION TITLE
Return result iteratively in chunks

### DISCUSSION THREAD

**Discussion by wangxiaoying** [sfu-db/connector-x#63] [Ideas]
Functionality refer to [pandas.read_sql](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_sql.html) with `chunksize` specified.

**Comment by alabarga** [comment]
This would be great, not only for pandas, also for arrow

**Comment by alabarga** [comment]
This would be great, not only for pandas, also for arrow
